backend/Origam.Server/Configuration/IdentityGuiConfig.cs

```diff
@@ -21,10 +21,13 @@
 
 #endregion
 
+using Origam.Server.Authorization;
+
 namespace Origam.Server.Configuration
 {
     public class IdentityGuiConfig
     {
         public bool AllowPasswordReset { get; set; }
+        public bool IsInitialSetupNeeded => UserTools.IsInitialSetupNeeded();
     }
```

Comment:
I don't think it is a good idea to add to the configuration something that is not read from the config file. Could you use the UserTools.IsInitialSetupNeeded method directly?